core/src/unprocessed_packet_batches.rs

```diff
@@ -0,0 +1,1205 @@
+use {
```

Comment:
+1 for refactoring this, but I think this can be a separate PR that gets checked in first